test/integration/scheduler_perf/README.md

```diff
@@ -31,14 +31,39 @@ Currently the test suite has the following:
 
 How To Run
 ------
+
+## Density tests
+
 ```shell
 # In Kubernetes root path
-make generated_files
+make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=true -logtostderr=true -run=." KUBE_TIMEOUT="--timeout=60m" SHORT="--short=false"
+```
 
-cd test/integration/scheduler_perf
-./test-performance.sh
+## Benchmark tests
+
+```shell
+# In Kubernetes root path
+make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling"
```

Comment:
what does benchtime=1ns mean? should we clarify this option in the doc?

Reply:
Benchtime is set to 1ns so the golang benchmark iterates at most once. Yeah, we might mention why it's explicitly set to 1ns instead of the default 1s.

Reply:
Done